omx/gstomxaacdec.c

```diff
@@ -232,6 +275,7 @@ gst_omx_aac_dec_is_format_change (GstOMXAudioDec * dec, GstOMXPort * port,
 static gint
 gst_omx_aac_dec_get_samples_per_frame (GstOMXAudioDec * dec, GstOMXPort * port)
 {
+  g_return_val_if_fail (dec, GST_FLOW_ERROR);
```

Comment:
`GST_FLOW_ERROR` has no special meaning in this context. The receiver will interpret it as a valid integer and use it. I recommend an invalid value, such as -1 or 0.